docs/source/quadrature.rst

```diff
@@ -0,0 +1,113 @@
+.. only:: html
+
+  .. contents::
+
+Quadrature rules
+================
+
+To numerically compute the integrals in the variational formulation,
+a quadrature rule is required.
+By default, Firedrake obtains a quadrature rule by estimating the polynomial
+degree of the integrands within a :py:class:`ufl.Form`. Sometimes
+this estimate might be quite large, and a warning like this one will be raised:
+
+.. code-block::
+
+   tsfc:WARNING Estimated quadrature degree 13 more than tenfold greater than any argument/coefficient degree (max 1)
+
+Sometimes the estimated quadrature degree might be even in the hundreds or thousands,
+rendering the integration prohibitively expensive. 
+
+Specifying the quadrature rule in the variational formulation
+-------------------------------------------------------------
+
+To manually override the default, the
+quadrature degree can be prescribed on each integral :py:class:`~ufl.measure.Measure`,
+
+.. code-block:: python3
+
+   inner(sin(u)**4, v) * dx(degree=4)
+
+Setting ``degree=4`` means that the quadrature rule will be exact only for integrands
+of total polynomial degree up to 4. This, of course, will introduce a greater numerical error than the default.
+
+The default quadrature degree estimation may be overridden by setting the ``"quadrature_degree"`` in
```

Comment:
Earlier you say "to manually override the default". It's not clear how this ("the default ... may be overridden") is different.